# SOG infile patch to produce a testcase infile from the repository infile

# This patch is used by buildbot.

# It was created by:
#    $ cd SOG
#    $ cat tests/infile_patches/patch_comments > path_to/patch_file
#    $ diff -u infile path_to/sample_new_infile >> path_to/patch_file

#  To apply this patch manually, do:
#    $ cd SOG
#    $ patch -i path_to/patch_file -o path_to/new_infile

# where:
#   - tests/infile_patches/patch_comments contains these comments
#   - path_to/patch_file is the path and name of this file
#   - path_to/new_infile is the path and the name of the new infile to be
#     created.

--- infile	2009-07-19 10:19:56.000000000 -0700
+++ infiles/R3no_remineralization	2009-07-19 16:09:36.000000000 -0700
@@ -2,10 +2,11 @@
 ! $Id$
 ! $Source$
 
-! SOG code infile template
-
-! SOG code infile for 356.5 day run starting at cruise 04-14 station S3
-! CTD cast (2004-10-19 12:22 LST).
+! SOG code infile for ~1 yr run starting at cruise 03-20 station S3
+! CTD cast.  This is the R3 no-remineralization regression test
+! infile.  Flagellates, remineralization, and microzooplankton are all
+! disabled in the biology model, and single species light limitation
+! is enabled.
 
 ! Grid parameters:
 "maxdepth"	40.d0	"depth of modelled domain [m]"
@@ -14,9 +15,9 @@
 
 ! Run timing parameters:
 ! Initial conditions date/time (must match a CTD profile date/time)
-"init datetime"	"2004-10-19 12:22:00"	"initialization CTD profile date/time"
+"init datetime"	"2003-10-09 11:47:00"	"initialization CTD profile date/time"
 ! End of run date/time
-"end datetime"	"2005-10-11 00:22:00"	"end of run date/time"
+"end datetime"	"2004-11-01 12:00:00"	"end of run date/time"
 "dt"		900	 "time step [s]"
 ! Implicit solver iteration limit
 "max_iter"	30	"implicit solver max iterations"
@@ -29,11 +30,11 @@
 
 ! Initial conditions
 "ctd_in"
-  "../sog-initial/ctd/SG-S3-2004-10-19.sog"
+  "../sog-initial/ctd/SG-S3-2003-10-09.sog"
   "initialization CTD file"
 ! Give a path/filename for either nuts_in or botl_in; set the other to "N/A"
 "nuts_in"
-  "../sog-initial/stratogem_nuts/Nuts-S3-2004-10-19.sog"
+  "../sog-initial/stratogem_nuts/Nuts-S3-2003-10-09.sog"
   "pre-processed nitrate & silicon profile data from STRATOGEM bottles"
 "botl_in"
   "N/A"
@@ -44,29 +45,29 @@
 
 ! Time series output files
 "std_phys_ts_out"  
-	"timeseries/std_phys_sog-test1.out"  
+	"timeseries/std_phys_R3noremin.out"  
 	"standard physics timeseries"
 "user_phys_ts_out" 
-	"timeseries/user_phys_sog-test1.out" 
+	"timeseries/user_phys_R3noremin.out" 
 	"user-defined physics timeseries"
 "std_bio_ts_out"   
-	"timeseries/std_bio_sog-test1.out"   
+	"timeseries/std_bio_R3noremin.out"   
 	"standard biology timeseries"
 "user_bio_ts_out"
-	"timeseries/user_bio_sog-test1.out"  
+	"timeseries/user_bio_R3noremin.out"  
 	"user-defined biology timeseries"
 
 ! Profiles output
 "noprof"	1		"no. of profiles to print"
 ! *** It would be nice to replace the yr-day and day-sec lists with a list
 ! *** of dates/times for profiles output
-"profday"	303		"yr-day for profile"
-"proftime"	43200.		"day-sec for profile"
-"haloclinefile"	"profiles/halo-sog-test1.out"	"file for halocline results"
-"profile_base"	"profiles/sog-test1"	"profile file base (datetime will be added)"
+"profday"	077		"yr-day for profile"
+"proftime"	48480.		"day-sec for profile"
+"haloclinefile"	"profiles/halo-R3noremin.out"	"file for halocline results"
+"profile_base"	"profiles/R3noremin"	"profile file base (datetime will be added)"
 ! Hoffmueller diagram output (a collection of profiles at time intervals
 ! for contour or colourmap plotting)
-"Hoffmueller file"	"profiles/hoff-sog-test1.dat"	"file for Hoffmueller results"
+"Hoffmueller file"	"profiles/hoff-R3noremin.dat"	"file for Hoffmueller results"
 "Hoffmueller start yr"	2004			"yr to start Hoff output"
 "Hoffmueller start day"	294			"yr-day to start Hoff output"
 "Hoffmueller start sec"	43200			"day-sec to start (midnight=0)"
@@ -136,10 +137,10 @@
 
 ! Biology model parameters:
 ! Model complexity flags:
-"flagellates_on"	.true.	"include flagellates in model?"
-"remineralization"	.true.	"include remineralization loop in model?"
-"use microzooplankton"  .true.  "include microzooplankton in model?"
-"single species light"  .false.  "high light limitation for single species case"
+"flagellates_on"	.false.	"include flagellates in model?"
+"remineralization"	.false.	"include remineralization loop in model?"
+"use microzooplankton"  .false.  "include microzooplankton in model?"
+"single species light"  .true.  "high light limitation for single species case"
 ! Zooplankton values:
 ! tuned
 "Mesozoo, winter conc"       0.1d0 "Mesozoo background concentration"
@@ -389,7 +390,7 @@
 "Englishman integ days"  20       "Days over which to integrate Englishman River"
 
 ! Start year to run model: used in forcing.f90
-"startyear"     2004    "initialization of runtime for model"
+"startyear"     2003    "initialization of runtime for model"
 
 ! File names for forcing data (wind, met, rivers)
 "Wind"
